ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -168,7 +159,7 @@ open class Tooltip: NSObject, TokenizedControlInternal {
             tooltipView.alpha = 0.0
         }, completion: { _ in
             tooltipView.removeFromSuperview()
-            UIAccessibility.post(notification: .screenChanged, argument: self.tooltipViewController?.positionController.anchorView)
+            UIAccessibility.post(notification: .screenChanged, argument: self.anchorView)
```

Comment:
to be defensive, we should always capture [weak self] in the completion blocks

` completion: { [weak self] in
                guard let strongSelf = self else {
                    return
                }
strongSelf.foo
`